var/spack/repos/builtin/packages/python/package.py

```diff
@@ -254,7 +254,6 @@ class Python(Package):
     variant("ssl", default=True, description="Build ssl module")
     variant("sqlite3", default=True, description="Build sqlite3 module")
     variant("dbm", default=True, description="Build dbm module")
-    variant("nis", default=False, description="Build nis module")
```

Comment:
This still exists until 3.13. I think it is probably better to add a `when="@:3.13"` here for now and we can remove it later when that version is dropped.

Reply:
The issue is that you cannot disable nis

Reply:
Actually you can disable nis, I've submitted https://github.com/spack/spack/pull/47862.